embedded-graphics/src/style/text_style.rs

```diff
@@ -35,19 +42,100 @@ where
             background_color: None,
         }
     }
+
+    /// Create a text style with only the font defined
```

Comment:
Is this function necessary? It can easily be replaced by `TextStyleBuilder::new(font).build()` and `TextStyle`s without any set colors are rarely useful.

Reply:
Good point. Removed.